content/en/wiki-intro.md

```diff
@@ -6,13 +6,13 @@ sidebar_label: Introduction
 
 ![Darwinia Banner](assets/darwinia-banner.png)
 
-Darwinia Network is a decentralized cross-chain bridge network building on Substrate, which is the "Golden Gate Bridge" of the cross-chain ecology. It provides the safest general bridge solution, connecting Polkadot, Ethereum, TRON and other heterogeneous chains by cross-chain assets transfer and general remote chain call. Also, its main application areas include Defi, cross-chain NFT trading market, games, etc.
+Darwinia Network is a decentralized cross-chain bridge network building on Substrate, which is the "Golden Gate Bridge" of the cross-chain ecology. It provides a safe and general bridging solution, connects to Polkadot, Ethereum, TRON and other heterogeneous chains by cross-chain transfer of assets and remote chain calls. The application areas of Darwinia Network include DeFi, cross-chain NFT trading, games, etc.
 
-Darwinia Network has won a high reputation and recognition along the way to build the decentralized cross-chain bridge protocol. In 2020, Darwinia was written in Polkadot light-paper as one of the friends of Polkadot and Substrate. And Darwinia was also selected to joined Substrate Builder Program and Web3.0 Bootcamp, and in recognition of Darwinia Network’s outstanding performance in the Substrate Builder Program, Parity officially awarded the Level 2 badge to Darwinia.Products and tools developed by Darwinia won 3 W3F Grants. Polkadot Founder Gavin also highlighted Darwinia as “Projects to watch.”
+Darwinia Network has won a high reputation and recognition along the way to build the decentralized cross-chain bridge protocol. In 2020, Darwinia was written in Polkadot light-paper as one of the friends of Polkadot and Substrate. And Darwinia was selected to join Substrate Builder Program and Web3.0 Bootcamp, and for the outstanding work in Substrate Builder Program, Darwinia Network was officially awarded the Level 2 badge by Parity. Products and tools developed by Darwinia have won three W3F Grants. Polkadot Founder Gavin Wood also highlighted Darwinia Network as “Projects to watch.”
```

Comment:
1. `won` -> `gained` 
2. `Polkadot Founder` -> `Polkadot founder`
3. `Products and tools` -> `The products and tools`

A few tips on this document.